chronosphere/notification_policy_data.go

```diff
@@ -48,10 +48,12 @@ type Routes struct {
 }
 
 // RoutesNotifierList is a list of notifiers along with configuration for
-// how when those notifiers should repeat alerting.
+// when those notifiers should repeat alerting and whether they should be grouped
+// by any specific values.
 type RoutesNotifierList struct {
 	Notifiers      []*RoutesNotifierListNotifier `json:"notifiers"`
 	RepeatInterval string                        `json:"repeat_interval,omitempty"`
+	GroupBy        *RoutesNotifierListGroupBy    `json:"group_by,omitempty"`
```

Comment:
can you update one of the examples in the examples/ directory to include use of group_by

can update an existing resource or create a new one

Reply:
done!